config/pitest-suppressions/pitest-api-suppressions.xml

```diff
@@ -1,14 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressedMutations>
-  <mutation unstable="false">
-    <sourceFile>FileText.java</sourceFile>
-    <mutatedClass>com.puppycrawl.tools.checkstyle.api.FileText</mutatedClass>
-    <mutatedMethod>&lt;init&gt;</mutatedMethod>
-    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.MemberVariableMutator</mutator>
-    <description>Removed assignment to member variable charset</description>
-    <lineContent>charset = null;</lineContent>
-  </mutation>
-
```

Comment:
This is an equivalent mutation according to pitest docs. See https://pitest.org/quickstart/mutators/#EXPERIMENTAL_MEMBER_VARIABLE.

This mutator is replacing:
```java
charset = null;
```
with 
```java
charset = null;
```
Yes, it is not doing anything, the code isn't actually mutated. Pitest blindly replaces member variable values with default values. So no test would be able to kill this mutation. 

I opened https://github.com/hcoles/pitest/issues/1230 for this.

@romani @rdiachenko Please share your thoughts on this one.

Reply:
We can estimate how much value this mutator  introduced. If we fixed a lot from it already and this is just rare case of unfortunate behavior, we can keep special issue in our project (linked to pitest) to wait pitest fix and keep them in survivals.

If almost all are same problem, we can deactivate whole mutator in pom.xml .

@Vyom-Yadav , what is your opinion on this?

Reply:
@romani There are 114 mutations from this mutator right now:
```bash
$ grep -ir "MemberVariableMutator" config/pitest-suppressions | sed '/^\s*$/d' | wc -l
114
```

I manually went through a few mutations to see the value this introduces. From what I saw, there is a mix of actual and false-positive mutations. I would vote for not-deactivating this mutator as it has significant cases that carry value. Let's trust pitest maintainers to fix this issue like they previously did at https://github.com/hcoles/pitest/issues/1066

Reply:
I have no worries that they will fix it, I just try to think on how to postpone some resolution to the end in hope that fix can be done in next 1-3 months.

Reply:
@Kevin222004 , as you go through survivals please mark them in issue description by ` delayed until hcoles/pitest#1066` to make it clear that you looked at it and think that better to not do it.

Reply:
ok, will update issue description as i went through this type of mutation